community/events/sprints.md

```diff
@@ -0,0 +1,365 @@
+
+To add contributors to the repository using the All-Contributors bot:
+
+**Use the Bot Command**: In a comment on an issue or pull request, use the following command to add a contributor as follows.
+
+  `@all-contributors add @githubusername for <contribution types>`
+
+If they have opened an issue only, or reviewed an open pr use:
+
+  `@all-contributors add @githubusername for review`
+
+If they have opened a pull request use:
+  `@all-contributors add @githubusername for code, review`
+
+
+2. **Merge all-contributor bot Pull Requests Individually**: The bot will create a pull request to update the
+contributors list when you call the commands above. IMPORTANT: Merge each all-Contributors bot pull request individually and immediately before adding another contributor to avoid merge conflicts.
+
+By recognizing contributors for their efforts, we foster a positive and inclusive
+community, encouraging more participation and collaboration.
+
+## After a Sprint tasks
+
+### Issue and PR triage
+The biggest effort after a sprint will be triaging & addressing issues and pull requests that have been submitted during the sprint. This could take 1-2 weeks and up to a month depending on the scope of each change submitted.
+
+If there are new issues with no associated pull request, you can
+
+1. Invite the issue author to submit a pr if the change seems reasonable and they are up to the task.
+2. If the issue is one that we agree should be addressed, then label the issue with `help-wanted` / `sprintable`  so someone else can tackle it at our next pyOpenSci sprint.
+
+Review each pull request submitted (or invite community members to review) and determine if the pull request looks good as is or requires changes following the standard GitHub review process.
+
+Merge the pull request when it is complete and ready to be merged.
+
+### Followup thank you notes
+
+After a sprint, we will follow up with participants to show our appreciation
+and encourage continued engagement with the pyOpenSci community. Here are the key
+steps to take:
+
+1. **Send a Thank You Email**: Send an email to all sprint participants thanking them for their contributions. Express gratitude for their time and effort, highlighting any significant achievements or milestones reached during the sprint.
+
+2. **Provide Feedback**: Include any relevant feedback or outcomes from the sprint. This could involve sharing metrics, such as the number of issues closed, pull requests merged, or any specific contributions that stood out.
+
+3. **Invite Continued Engagement**:
+   - **Newsletter**: Encourage participants to subscribe to the pyOpenSci newsletter
+     to stay updated on future events, news, and opportunities.
+   - **LinkedIn**: Invite them to follow pyOpenSci on LinkedIn for professional updates
+     and community highlights.
+   - **GitHub**: Remind them to continue following and contributing to pyOpenSci
+     repositories on GitHub.
+    - Discourse?
```

Comment:
reading my mind! I have a note to talk about Discourse in our upcoming social media quarterly meeting.

Reply:
cool. how bout i just comment this out for the time being?

Reply:
perfect!